### PR TITLE
Add option to pass abort signal

### PR DESCRIPTION
**Pull request by sobstel**
We sometimes (eg when user logs out while requests are being made) need to abort fetch requests. This PR adds possibility to use abort signal (returned by AbortController), so it can be handled without hacks.

### CODE REVIEW

**Review by michax — CHANGES_REQUESTED** [review]

File: src/utils/fetch.js [hunk 1]
```diff
@@ -253,7 +256,7 @@ function localFetch(method, url, options) {
         });
     }, 30000);
 
-    return fetch(url, { headers, method, body, credentials })
+    return fetch(url, { headers, method, body, credentials, signal })
```
[the remark this answers is not in the document]
Comment (sobstel) [reply src/utils/fetch.js:259]: `fetch` is not supported by IE11 at all

[the remark this answers is not in the document]
Comment (michax) [reply src/utils/fetch.js:259]: Quite vague comment on my end, quite rushed, sorry.
Anyhow, all my worries regarding IE11, polyfill etc can be solved on library consumer side.

**Review by michax — APPROVED** [review]

File: src/openapi/transport/core.spec.js [hunk 2]
```diff
@@ -865,4 +865,32 @@ describe('openapi TransportCore', () => {
             fetch.mockClear();
         });
     });
+
+    describe('abort signal', () => {
```
Comment (michax) [inline src/openapi/transport/core.spec.js:869]: Nitpick. Let's fix prettier errors.

image

**Review by michax — CHANGES_REQUESTED** [review]

File: src/openapi/transport/core.spec.js (hunk 2 above)
Comment (michax) [reply src/openapi/transport/core.spec.js:869]: Resolved it as part of preparing beta build/

**Review by michax — APPROVED** [review]

**Pull request merged**